semgrep-core/src/runner/Runner_config.ml

```diff
@@ -9,6 +9,16 @@ open Common
  * found by extract rules.
  *)
 type output_format = Text | Json of bool (* dots *) [@@deriving show]
+type rule_source = Rule_file of filename | Rules of Rule.t list
```

Comment:
maybe comment on what is usually used when using semgrep-core directly (rule_file), and
one when passed by semgrep-ocaml?

Reply:
I added comments.